site/quickstart.md

```diff
@@ -1,159 +1,177 @@
+```
+$ kubectl create namespace sample-domains-ns1
+```
+b.	Use `helm` to configure the operator to manage domains in this namespace:
 
-```  
-$ docker pull store/oracle/weblogic:12.2.1.3-dev
 ```
- * For reference, see the [domain home on PV README](https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md).
- * The `create-domain.sh` will:
-   * Create a directory for the generated Kubernetes YAML files for this domain. The pathname is `/path/to/weblogic-operator-output-directory/weblogic-domains/`.
-   * Create a Kubernetes job that will start up a utility WebLogic Server container and run offline WLST scripts, or WebLogic Deploy Tool (WDT) scripts, to create the domain on the shared storage.
-   * Run and wait for the job to finish.
-   * Create a Kubernetes domain resource YAML file, `domain-custom-resource.yaml`, in the directory that is created above. You can use this YAML file to create the Kubernetes resource using the `kubectl create -f` or `kubectl apply -f` command:
+$ helm upgrade \
+  --reuse-values \
+  --set "domainNamespaces={sample-domains-ns1}" \
+  --wait \
+  sample-weblogic-operator \
+  kubernetes/charts/weblogic-operator
 
 ```
-          ./create-domain.sh
-          -i create-domain-inputs.yaml
-          -o /path/to/output-directory
+c.  Configure Traefik to manage Ingresses created in this namespace:
+```
+$ helm upgrade \
+  --reuse-values \
+  --set something about namespaces
 ```
 
-* For a domain home in image, use the sample in the Docker GitHub project.
-
-* Optionally, create a configuration overrides template and any additional Kubernetes secrets it needs (for example, to override the domain home configuration of a database URL, username, and password).
-
-* Create a domain resource in the domain namespace.
-  * Specify the following information: domain UID, service account, secret name, the domain home details, and optionally, the configuration overrides template name.
+## 5. Create a domain in the domain namespace.
 
-* Configure the operator to know about the domain.
-   * Edit the operator `values.yaml` file to add the namespace of the domain:
+a.	Create a new image with a domain home by running the [`create-domain-in-image`](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md) script.
+* First, create a directory for the files generated by the sample.
+* Then, copy the `inputs.yaml` file from the sample and modify it:
+  * Specify a `domainUID`: `sample-domain1`
+  * Specify the domain namespace: `sample-domains-ns1`
 
+b.	The sample also creates a domain resource YAML file.
+* Also, use `kubectl apply -f` on the YAML file to create the domain resource:
 ```
-$ helm update kubernetes/charts/weblogic-operator --name my-operator --namespace weblogic-operator-ns --values values.yaml --wait
+kubectl apply -f /path/to/output-directory/weblogic-domains/sample-domain1/domain.yaml
```

Comment:
"/path/to/output-directory" is the directory that is created in the first step. In step b. the domain resource YAML file is generated in </path/to/outpit-directory>/weblogic-domains/sample-domain1.

Reply:
I will point them to the domain-home-in-image/README.md and just make sure that they have all the specific inputs they need (like the domain UID, the domain namespace and how to "reate the Kubernetes secrets username and password of the admin account in the same Kubernetes namespace as the domain."